src/os/bluestore/bluefs_types.cc

```diff
@@ -5,23 +5,24 @@
 #include "common/Formatter.h"
 #include "include/uuid.h"
 #include "include/stringify.h"
+#include "include/small_encoding.h"
 
 // bluefs_extent_t
 
 void bluefs_extent_t::encode(bufferlist& bl) const
 {
   ENCODE_START(1, 1, bl);
```

Comment:
Seems bluestore is far from stable and production ready and we are safe not to bother do this for now :-(   Besides, recently we have changed bluestore on-disk types a lot and we don't raise struct/compat version either.